### PR TITLE
Add BUILD_STATUS.md to Chrobalt.

### PR DESCRIPTION
**Pull request by briantting**

### CODE REVIEW

File: BUILD_STATUS.md
Comment (oxve) [inline BUILD_STATUS.md:1]: I think we should remove broken and obsolete entries. Rows that won't have any badges (workflows that only ran on `main`) we should remove.

The rich diff shows what's broken: https://github.com/youtube/cobalt/pull/4192/files?short_path=c6f3fe6#diff-c6f3fe6286b445ee90f9290fd0470c993b0834f1d558b9abd7beadc98f5eeb20

Comment (briantting) [reply BUILD_STATUS.md:1]: done.

File: BUILD_STATUS.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+| Xbox One  | [![xb1](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml?query=event%3Apush+branch%3Amain) | | | | | | | | |
```
Comment (oxve) [inline BUILD_STATUS.md:11]: win32 and xb1 can be removed for trunk. They haven't run in a while. Same below.

**Review by oxve — APPROVED** [review]

**Review by dahlstrom-g — COMMENTED** [review]
A few questions about this PR:
* Do we have to add this file in the root?
* Why are the contents so different from before? https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1
* Isn't it unhygienic to name various branches in a file on the `main` branch?
* Why does it have 7 commits?

File: BUILD_STATUS.md
Comment (oxve) [inline BUILD_STATUS.md:1]: @kaidokert Made a good point about this file, it should probably live under the `chrobalt` directory instead.

**Comment by oxve** [comment]
> A few questions about this PR:
> 
> * Do we have to add this file in the root?

It should live under `/chrobalt`.


> * Why are the contents so different from before? [`pre-chrobalt`/BUILD_STATUS.md?plain=1](https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1&rgh-link-date=2024-10-03T02%3A49%3A42Z)

We're building very different things now. See my comment earlier on the PR.

> * Isn't it unhygienic to name various branches in a file on the `main` branch?

This is the overall build status, it should show all the important builds in one place.

**Comment by briantting** [comment]
> A few questions about this PR:
> 
> * Do we have to add this file in the root?
> * Why are the contents so different from before? https://github.com/youtube/cobalt/blob/pre-chrobalt/BUILD_STATUS.md?plain=1
> * Isn't it unhygienic to name various branches in a file on the `main` branch?
> * Why does it have 7 commits?

Added to Chrobalt instead, because added 25 lts and cleaned up main which only does linux and android now, we want this to be a singular status page to monitor everything so kinda but for ease of use its necessary and what we were doing before, because when i hit edit within github thats what happens (its 8 now)

**Pull request merged**